threescale_api/resources.py

```diff
@@ -151,6 +151,12 @@ def __init__(self, *args, entity_name='account_plan', entity_collection='plans',
     def url(self) -> str:
         return self.threescale_client.admin_api_url + '/account_plans'
 
+    def create(self, params: dict = None, **kwargs):
+        url = self.url
+        response = self.rest.post(url=url, json=params, **kwargs)
+        instance = self._create_instance(response=response)
+        return instance
+
```

Comment:
@JaurbanRH Why did you introduce this method?

If I am not mistaken there is already `create` method in the `DefaultClient` class.
https://github.com/pestanko/3scale-api-python/blob/master/threescale_api/defaults.py#L74